snapshot/policy/actions_policy.go

```diff
@@ -11,12 +11,16 @@ type ActionsPolicy struct {
 	// commands run once before and after each snapshot root (can be inherited).
 	BeforeSnapshotRoot *ActionCommand `json:"beforeSnapshotRoot,omitempty"`
 	AfterSnapshotRoot  *ActionCommand `json:"afterSnapshotRoot,omitempty"`
+
+	// use Windows Volume Shadow Copy Service (can be inherited).
+	VSS *OptionalBool `json:"vss,omitempty"`
```

Comment:
it would be good to consider future direction where we may want to add first-class support for ZFS, LVM and other types of snapshots (some of which may need more than a simple boolean to enabled/disable them).

I would recommend making this more generic and possible to add parameters, and perhaps create a whole new section for OS-specific point-in-time snapshotting of filesystems and block devices, like so:

```
"osSnapshot": {
  "volumeShadowCopy": {
    "enabled": true,
    // any VSS parameters go here.
  },
  "zfs": {
   "enabled": true,
   "mountPath": "..."
  },
  "lvm": {
   "enabled": true,
  },
  "btrsfs": {
   "enabled": true,
  }
},
```

This may be nested in `actions` or parallel to it, not sure which one is better.